app/src/main/java/com/monkeyteam/chimpagne/ui/components/Button.kt

```diff
@@ -83,7 +84,7 @@ fun IconTextButton(
               .padding(horizontal = 24.dp, vertical = 12.dp)) {
         Icon(icon, contentDescription = text)
         Spacer(Modifier.width(8.dp))
-        Text(text)
+        Text(text.uppercase(Locale.ROOT))
```

Comment:
This is a very welcome addition, and ensures consistency of look and feel going forward 😎